service/ui.go

```diff
@@ -292,6 +292,15 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 		))
 	}
 
+	// Lobster
+	app.PrefixRoute("/lobster").Route("/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
+	app.PrefixRoute("/lobster").Route("/{*}/{*}/{*}/{*}/{*}/{*}/").Handler(uis.lobsterPage).Get()
```

Comment:
Do you know how the correct base URLs to fetch the raw logs get populated? I see that there are REACT_APP_LOGKEEPER_BASE and REACT_APP_EVERGREEN_BASE env vars but I don't see that logkeeper ever used these

Reply:
the base urls are set here: https://github.com/SupaJoon/lobster/blob/main/src/config.js#L5

Reply:
Right but the logkeeper one defaults to a blank string, and logkeeper/evergreen's current build system doesn't set the environment var from what I can tell

Reply:
ah that's right. The blank string used to make sense when lobster was hosted in logkeeper. We can export environment variables in this [ticket](https://jira.mongodb.org/browse/EVG-14179) before [this line](https://github.com/evergreen-ci/logkeeper/blob/main/buildscripts/update-lobster.sh#L37)

Reply:
I remember that will be no longer serving logs from logkeeper some day so the logkeeper api route could be deprecated by the time we release the log viewing experience to our users. Does it make sense omit this change and remove the Logkeeper base url and relevant code paths from Lobster?